### ISSUE TITLE
Dumping at L&Q site

### ISSUE THREAD

**Issue by theparksilford** [theparksilford/Issues#16]
### Property Details

25 to 35 Lawson close

### Contact Details

Internal

### Type of problem?

General Issue

### Since when this issue started?

5 Jan 2021

### Issue Description?

We are seeing lots of dumping at L&Q site. Recently The occupant of the middle floor flat on the extreme left of the block 25 to 35 Lawson Close with a red bicycle on the balcony - he about 35 to 45  full black head of hair and his son 10 to 13 have just dumped a complete bed frame plus maybe other items in the communal bin store at the top of Lawson Close immediately next to their flat. 
we need to ask L&Q to take action on it and get this clear. If they don’t we will charge them for cleaning that area.  Ask then to visit their site and let residents know about the restrictions. 

### Logged By

Leaseholder or FreeHolder

**Comment by theparksilford** [comment]
Also this is by the other bin at L&Q location. 
![Uploading 2FDF2AB9-5A3E-424A-B703-6BAF1BA3BC44.jpeg…]()

**Comment by theparksilford** [comment]
Also this is by the other bin at L&Q location. 
![F20DB9BB-C041-4AC3-B788-AE539E1F02E4](https://user-images.githubusercontent.com/85579935/148580447-e7e4d852-40fe-4816-a891-36680cf723dc.jpeg)

**Comment by jeremydoran** [comment]
Thank you for letting me know.

L&Q have now been contacted about this and I await a response / action to this matter.

**Comment by theparksilford** [comment]
![04E67967-BE7B-49D8-B2D7-EC72A39D8D9C](https://user-images.githubusercontent.com/85579935/148985205-2c737e4a-093f-4b05-a7d2-ea66b3b741bc.jpeg)
I also have a video of their resident throwing some stuff. They came from their balcony so it’s straight forward for them to figure out. Let me know if they want it. 
If they want us to clear for them then we will just charge them whatever it costs to clear and put this in their service charge if they want to, otherwise clear these stuff themselves please.

**Comment by theparksilford** [comment]
Today Council didn’t collect the bins from that bin area because they can’t access it. Have we heard anything from L&Q?

![1B7B51D6-7CF6-4626-B11A-21D5E377F95A](https://user-images.githubusercontent.com/85579935/149358835-771f1b3f-5b77-4422-9afb-87c580f7fa30.jpeg)

**Comment by jeremydoran** [comment]
I have contacted L&Q again about this as I haven't received any response regarding this issue.

L&Q are notoriously difficult to deal with and to communicate with.

**Comment by theparksilford** [comment]
![815F8C95-9BA9-4A4F-81BD-10008E302188](https://user-images.githubusercontent.com/85579935/149799599-b7cc881b-5a8e-439f-9aa1-fb5f21672c31.jpeg)
Any news this is getting bad! Could we not simply call them?. I suppose we can get it cleared and charge them money?

**Comment by jeremydoran** [comment]
I have emailed L&Q again, I did get a response previously to ask if it was L&Q responsibility in which I responded to advise it was. I have just sent another chaser email. 

I do not have a telephone number for the person I am emailing and it is very difficult to speak to anyone at L&Q. I am hoping this will be picked up and I will get a response soon.

**Comment by theparksilford** [comment]
ok, Please keep up the pressure. We can get it cleared for them, we charge whatever the cleaning company would charge in their Service Charge Account, they just need to agree on it or get it clear themselves and make their residents aware about the issue.

**Comment by jeremydoran** [comment]
L&Q have now confirmed that this has been raised, however they have advised that there is a turnaround of 20 business days.

**Comment by theparksilford** [comment]
There has been 16 days to open this issue. If they going to take 20 more days while both bins are not accessible and their residents are just piling refused outside this will become health and safely issue if it’s not already. Please communicate and ask them if they cannot pick this up in 2 days we will pick it up and charge their service account.

**Comment by jeremydoran** [comment]
I have now contacted L&Q about this and await a response.

**Comment by jeremydoran** [comment]
L&Q have now been in touch to advise that they have spoken to the contactor and they have scheduled this to take place this week.

**Comment by theparksilford** [comment]
looks like some things have been removed (thought I’d didn’t improve much). The other bin area dumping is not touched so not sure if it’s L&Q contractor who did it. I have also tweeted to Redbridge to act on it. Please keep the pressure. 
![CB66DADE-B027-4705-B31E-D17D14643ACA](https://user-images.githubusercontent.com/85579935/150835236-107e5652-0723-4bfb-baa4-9d1b2c9da611.jpeg)

**Comment by theparksilford** [comment]
Ok good news is that L&Q finally got their contractor cleared the mess from one bin and it looks clean. Council will come pickup the bins so that good but the other bin area is not and contractors said they weren’t told. 
Please ping L&Q for the other one too. 

![49F727C3-9507-473F-9BBF-CD098E223EA2](https://user-images.githubusercontent.com/85579935/150963428-13e71bef-d1f9-449e-bad0-4893101b742d.jpeg)
![E374B22B-939A-4610-88DB-74C033A02F4A](https://user-images.githubusercontent.com/85579935/150963447-94cb9d41-4610-4cb8-9799-92d4077b12a4.jpeg)

**Comment by jeremydoran** [comment]
I have just followed this up with L&Q as only some of the items have been removed.

**Comment by jeremydoran** [comment]
L&Q will be attending the development tomorrow and will follow up on the items that have yet to be removed.

**Comment by theparksilford** [comment]
This is still not resolved. The main bin was cleared however this second bin is still like this with sofa, fridge and other things lying there. Please ask them to get this cleared.

![75F8FE8E-5E73-49B5-AA77-BB4DCFCC456D](https://user-images.githubusercontent.com/85579935/152500350-7d128a38-5b08-4e62-9dfe-b6231dd6122e.jpeg)
![B7896B2C-565D-45C0-98EB-86D30821F79A](https://user-images.githubusercontent.com/85579935/152500381-76bc8be0-7d3a-4ea2-a3e7-2e9efe21612a.jpeg)
![D333AB61-D615-4035-99F9-3E1126AC0304](https://user-images.githubusercontent.com/85579935/152500388-54791ea8-42e4-419e-97d7-04437eb8b054.jpeg)

**Comment by jeremydoran** [comment]
I have followed this up with L&Q and sent over the pictures.

I will await a response.

**Comment by jeremydoran** [comment]
L&Q have responded to advise the bulk removal has been raised, just waiting on the contractor to schedule it in.

**Comment by theparksilford** [comment]
It’s finally cleared. I am sure the will happen again but we will create another issue again. Thanks closing this issue now.
